### DISCUSSION TITLE
Block country

### DISCUSSION THREAD

**Discussion by df911** [docker-mailserver/docker-mailserver#2928] [Q&A]
Hello,

I use mailserver in a docker stack and i would like to block many country (as wa can do with a Synology).

How to do that ?

I tried  to use ipset but I don't find documentation.

Thanks

**Comment by georglauterbach** [comment]
This is an issue tracker for feature requests and bug reports. I will be moving this into a discussion :)

EDIT: Done.

**Comment by df911** [comment]
Hello,

I think it will be a good idea to add geoip module into docker no ?

**Comment by mholti** [comment]
Hi,
just my two cents...
I think, this would be a mix of functionalty in one container, a mailserver is a mailserver, is a...
If you want to have geoblocking, why not putting a traefik-container in front of the mailserver and use a geoblocking middleware.
Short googeling showed: https://github.com/TheLastProject/SimpleGeoIPForwardAuth
Not tested here, but seems to be, what you are looking for.
